ui/src/actions/sampleGrid.js

```diff
@@ -103,6 +103,20 @@ export function syncSamples() {
   };
 }
 
+/**
+ * Sends a request to get the sample list and then synchronizes the samples with lims.
+ */
+export function sendGetandSyncSamples() {
```

Comment:
`send` prefix is now [reserved](https://mxcubeweb.readthedocs.io/en/latest/dev/front-end.html#fetching-layer) for API functions (in the `api` folder).

Since `sendGetandSyncSamples` is an action creator, you can just call it `getAndSyncSamples`.

Actually, it looks like that there are still a few action creators with the `send` prefix (including `sendGetSampleList`). I'll take care of it asap.